src/style.css

```diff
@@ -0,0 +1,22 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
```

Comment:
the last star shouldn't have margin-right: 4px;

Reply:
What should he have?